apps/libs/digest.h

```diff
@@ -0,0 +1,34 @@
+#ifndef TOCK_DIGEST_H
+#define TOCK_DIGEST_H
+
+#include <stdlib.h>
+
+// TODO These need to be standardized
+#define HOTEL_DRIVER_DIGEST 2
```

Comment:
Yes I agree 100%. As it stands, since there is really only one userspace, it's kind of de-facto standardized, but that won't be the case for very long. (a) we should actually do it systematically, (b) we should write it down somewhere.

I would suggest having low numbered system call major numbers be for very common drivers, in decreasing order of ubiquity/utility, and reserve higher numbered major numbers for more platform specific drivers.

For example, a mapping could be something like (large numbers are represented below as negative numbers):

```
0 => LEDs
1 => GPIOs
2 => Timer
3 => Console
...
9 => RNG
10 => SHA256
...
-3 => Platform specific driver
-2 => Platform specific driver
-1 => Platform specific driver
``` 

Worth you suggesting like this to the list?

(LEDs and GPIO drivers above are very similar, or even identical, but it makes sense to separate for userspace so an app doesn't have to know which pins are connected to LEDs --- might make sense to have the same for buttons, but this is all a bit besides the general point)